nominal/__init__.py

```diff
@@ -1,13 +1,48 @@
-import os
+from .nominal import (
+    set_default_connection,
+    get_default_connection,
+    upload_dataset_from_pandas,
+    upload_dataset_from_polars,
+    upload_dataset,
+    get_dataset_by_rid,
+    update_dataset,
+    create_run,
+    get_run_by_rid,
+    search_runs,
+    update_run,
+    add_dataset_to_run,
+    list_datasets_for_run,
+    add_attachment_to_run,
+    list_attachments_for_run,
+    upload_attachment,
+    get_attachment_by_rid,
+    update_attachment,
+    save_attachment,
+)
+from .sdk import Dataset, Run, Attachment, NominalClient
 
-# Allows:
-# import nominal as nm
-# nm.cloud ...
-# nm.data ...
-from . import cloud as cloud
-from . import data as data
-from .nominal import Dataset as Dataset
-from .nominal import Ingest as Ingest
-from .nominal import Run as Run
-
-os.environ["NOMINAL_BASE_URL"] = "https://api-staging.gov.nominal.io/api"
+__all__ = [
+    "set_default_connection",
+    "get_default_connection",
+    "upload_dataset_from_pandas",
+    "upload_dataset_from_polars",
+    "upload_dataset",
+    "get_dataset_by_rid",
+    "update_dataset",
+    "create_run",
+    "get_run_by_rid",
+    "search_runs",
+    "update_run",
+    "add_dataset_to_run",
```

Comment:
for the function api, should we go for a higher level abstraction? like, `create_run_csv` / `create_run_polars` that handles upload + create directly?

Reply:
I was shying away from this initially, primarily so that the function signatures remained small. I didn't love the idea of including disambiguating parameter names like `dataset_name`, `run_name`, etc so figured it was best to keep them separate. Also have a q here about usage that I'll ask you about directly

Reply:
for uploading from a run - we can probably just generate the name for the dataset, and remove the descriptions. then the signature is small, and they have more customizability through composing functions. but this keeps the barrier to entry super low.

Reply:
I'm going to add this, but I'll do that in a v1.1. Want to get this merged without much further new testing